ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -60,15 +60,12 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: azure-ipi-workers-rhel8-f28
-  cron: 6 9 3 * *
+- as: azure-ipi-workers-rhel8-f14
+  cron: 31 22 14,30 * *
   steps:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      FILTERS_ADDITIONAL: ~TELCO&
```

Comment:
Per https://github.com/openshift/release/pull/52588, this filter is still needed.